doc/ref/configuration/master.rst

```diff
@@ -5368,9 +5368,9 @@ and pkg modules.
 .. code-block:: yaml
 
     peer:
-      foo.example.com:
-          - test.*
-          - pkg.*
+      foo\.example\.com:
```

Comment:
This was previously documented without including escaping, could this possibly be considered a bug instead?

Reply:
It was just a docs inconsistency imho - the strings are interpreted as regular expressions, not globs. Since the ``.`` is valid for any symbol, that includes a dot and would have worked for a minion named ``foo.example.com`` - but the same would have been valid for ``foozexamplelcom`` etc.